core/templates/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -17,72 +17,88 @@
  * contributors to contribute.
  */
 
-require('domain/opportunity/ExplorationOpportunitySummaryObjectFactory.ts');
-require('domain/opportunity/SkillOpportunityObjectFactory.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import {
```

Comment:
@bansalnitish please ignore the code in contribution-opportunities-backend-api.service.ts. I had pushed my first two commits into the develop branch in my fork and because of that, they appeared here. I created this branch from develop so contribution-opportunities-backend-api.service.ts showed up here.

Reply:
I'm not sure what's going one here, are we expecting this to be a part of this PR?